frontends/web/src/api/lightning.ts

```diff
@@ -888,18 +887,21 @@ export const getNodeInfo = async (): Promise<NodeState> => {
 };
 
 export const getListPayments = async (params: ListPaymentsRequest): Promise<Payment[]> => {
-  return getApiResponse<Payment[]>(`lightning/list-payments?${qs.stringify(params, { skipNull: true })}`, 'Error calling getListPayments');
+  const queryString = new URLSearchParams(params as Record<string, any>).toString();
+  return getApiResponse<Payment[]>(`lightning/list-payments?${queryString}`, 'Error calling getListPayments');
 };
 
 export const getOpenChannelFee = async (params: OpenChannelFeeRequest): Promise<OpenChannelFeeResponse> => {
+  const queryString = new URLSearchParams(params as Record<string, any>).toString();
```

Comment:
Need to cast the type to keep TS happy as URLSearchParams only take in string values (not numbers, array etc). Another alternative is to convert all values to string explicitly.

Reply:
Looks like TypeScript only wants values to be string, URLSearchParams has no issues with values that are numbers but they are internally converted to string.

I'd slightly prefer if we don't make TS happy here and overwrite global URLSearchParams (in src/global.d.ts). But no strong opinion.

// global.d.ts
declare global {
  interface URLSearchParams {
    new (init?: Record<string, string | number>): URLSearchParams;
  }
}